apis/server/image_bridge.go

```diff
@@ -62,3 +64,17 @@ func (s *Server) searchImages(ctx context.Context, resp http.ResponseWriter, req
 	}
 	return json.NewEncoder(resp).Encode(response)
 }
+
+// removeImage deletes an image by reference
+func (s *Server) removeImage(ctx context.Context, resp http.ResponseWriter, req *http.Request) error {
+	name := mux.Vars(req)["name"]
+
+	option := &ctrd.ImageRemoveOption{}
```

Comment:
I think the "ImageRemoveOption" should be defined in mgr, not in ctrd.  mgr is the core control layer, the image_bridge layer can't not depend on ctrd directly.